src/easymotion/easymotion.ts

```diff
@@ -61,41 +61,41 @@ export class EasyMotion {
     var totalSteps = 0;
 
     if (length >= keyTable.length) {
-        var totalRemainder = Math.max(length - keyTable.length, 0);
-        totalSteps = Math.floor(totalRemainder / keyTable.length);
+      var totalRemainder = Math.max(length - keyTable.length, 0);
```

Comment:
I noticed all the formatting was wrong before last week, I imagine you just used the formatter on these files right?